hw/top_earlgrey/doc/design/README.md

```diff
@@ -404,6 +404,7 @@ For the purpose of `top_earlgrey`, the first option has been chosen to benefit s
 | otp_ctrl          | otp_ctrl      | 0x40130000 (core) |
 |                   |               | 0x40138000 (prim) |
 | lc_ctrl           | lc_ctrl       | 0x40140000 (regs) |
+|                   |               | 0 (dmi)           |
```

Comment:
It clearly doesn't matter, but maybe worth getting the "(dmi)" bit to line up with the other items? Maybe by expressing the address as `0x00000000`?

Reply:
I changed the address to 0x00000000 but it will stick there with the lcctrl.